torch/_subclasses/meta_utils.py

```diff
@@ -1571,6 +1579,12 @@ def is_c_of_r(complex_dtype, real_dtype):
             if t.is_parameter:
                 r._is_param = True
 
+            # See Note: [Creating symbolic nested int]
+            if t.nested_int is not None and not r.has_nested_int():
```

Comment:
hm how can `r.has_nested_int()` ever evaluate to `True` here? IIUC `r` was just created and this is the point at which we would set the symbolic nested int for it